mailpoet/lib/EmailEditor/Engine/theme.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://schemas.wp.org/trunk/theme.json",
-  "version": 2,
+  "version": 3,
```

Comment:
@costasovo You increased the theme's version here. Should we also increase this version in [our Core theme](https://github.com/mailpoet/mailpoet/blob/b510e07942af573162ccaf241418b8e53c16a6e3/mailpoet/lib/EmailEditor/Integrations/Core/theme.json)?

Reply:
Good point. I'll do it.